toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -0,0 +1,197 @@
+                        In 'build' mode it will use the latest published container.
+    VERISION        1.0 or 2.0. default: "2.0"
+    MOUNTS          mount a directory into the container. Should be of the form '/src/dir:/dest/dir'. 
+                    Can be specified multiple times.
+    ENABLE_REPO:    Set to 'y' to use local RPMs to satisfy package dependencies. default: "n"
+
+To see help, run 'sudo make containerized-rpmbuild-help'
+"
+}
+
+build_chroot() {
+    pushd "${repo_path}/toolkit"
+    echo "Building worker chroot..."
+    sudo make graph-cache REBUILD_TOOLS=y > /dev/null
+    popd
+}
+
+build_tools() {
+    pushd "${repo_path}/toolkit"
+    echo "Building tools..."
+    sudo make go-depsearch REBUILD_TOOLS=y > /dev/null
+    sudo make go-grapher REBUILD_TOOLS=y > /dev/null
+    sudo make go-specreader REBUILD_TOOLS=y > /dev/null
+    popd
+}
+
+build_graph() {
+    pushd "${repo_path}/toolkit"
+    echo "Building dependency graph..."
+    sudo make workplan > /dev/null
+    popd
+}
+
+while (( "$#")); do
+  case "$1" in
+    -m ) mode="$2"; shift 2 ;;
+    -v ) version="$2"; shift 2 ;;
+    -p ) repo_path="$(realpath $2)"; shift 2 ;;
+    --enable-local-repo ) enable_local_repo=true; shift ;;
+    --help ) help; exit 1 ;;
+    ? ) echo -e "ERROR: INVALID OPTION.\n\n"; help; exit 1 ;;
+  esac
+done
+
+# Assign default values
+[[ -z "${repo_path}" ]] && repo_path=$(realpath "$(dirname "$0")") && repo_path=${repo_path%'/toolkit'*}
+[[ ! -d "${repo_path}" ]] && { print_error " Directory ${repo_path} does not exist"; exit 1; }
+[[ -z "${mode}" ]] && mode="build"
+[[ -z  "${version}" ]] && version="2.0"
+
+echo "Running in ${mode} mode, requires root..."
+echo "Running as root!"
+cd "${script_dir}"  || { echo "ERROR: Could not change directory to ${script_dir}"; exit 1; }
+
+# ==================== Setup ====================
+build_dir="${script_dir}/build_container"
+mkdir -p "${build_dir}"
+
+# Get Mariner GitHub branch at $repo_path
+pushd ${repo_path}
+repo_branch=$(git rev-parse --abbrev-ref HEAD)
+popd
+
+#Set splash text based on mode
+if [[ "${mode}" == "build" ]]; then
+    splash_txt="builder_splash.txt"
+else
+    splash_txt="tester_splash.txt"
+fi
+
+# ============ Populate SRPMS ============
+# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container
+pushd "${repo_path}/toolkit"
+echo "Populating Intermediate SRPMs..."
+sudo make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
+popd
+
+# ============ Map chroot mount ============
+if [[ "${mode}" == "build" ]]; then
+    # Create a new directory and map it to chroot directory in container
+    if [ -d "${repo_path}/build/container-build" ]; then rm -Rf ${repo_path}/build/container-build; fi
+    if [ -d "${repo_path}/build/container-buildroot" ]; then rm -Rf ${repo_path}/build/container-buildroot; fi
+    mkdir ${repo_path}/build/container-build
```

Comment:
Should `${repo_path}/build/container-build` be place in a variable so you don't need to repeat it 4 separate times.